test-utils/store-validator/src/lib.rs

```diff
@@ -0,0 +1,120 @@
+use borsh::BorshDeserialize;
```

Comment:
can we keep everything related to validation in only file? two reasons:
1. validation functions are very related to the `StoreValidator` class, it's useful to keep them together
2. I will change most of the code anyway (at least in https://github.com/nearprotocol/nearcore/pull/2680) and make functions closer to the `StoreValidator` class

we can go this way later, if necessary

Reply:
My logic here is that we're going to have more validation functions potentially split into multiple files (would make sense to separate trie validation logic for instance)

Reply:
it makes sense to me
however, this is done too early: I'm going to explore @frol ideas about architecture and it may lead to major infrastructure updates
I guess it's better to stabilize interface first